cmd/crank/xpkg/build.go

```diff
@@ -36,10 +39,14 @@ import (
 )
 
 const (
-	errGetNameFromMeta = "failed to get package name from crossplane.yaml"
-	errBuildPackage    = "failed to build package"
-	errImageDigest     = "failed to get package digest"
-	errCreatePackage   = "failed to create package file"
+	errGetNameFromMeta                = "failed to get package name from crossplane.yaml"
+	errBuildPackage                   = "failed to build package"
+	errImageDigest                    = "failed to get package digest"
+	errCreatePackage                  = "failed to create package file"
+	errParseControllerImage           = "failed to parse controller image"
+	errPullControllerImage            = "failed to pull controller image"
+	errLoadControllerTar              = "failed to load controller tar"
+	errGettingControllerBaseImageOpts = "failed to get controller base image options"
```

Comment:
You’ve used `get` in the previous error, and `getting` here - probably nice to be consistent? 😊

Reply:
good catch 😉